tests/Resources/Page/OCPDashboard/Pods/Pods.robot

```diff
@@ -132,8 +132,21 @@ Verify Container Image
     Should Be Equal  ${image}  ${expected-image-url}
 
 Search Pod
-    [Documentation]   Return list of pods   pod_start_with = here ypu have to provide starting of pod name
+    [Documentation]   Returns list of pods   pod_start_with = here ypu have to provide starting of pod name
     [Arguments]   ${namespace}  ${pod_start_with}
     ${pod} =  Run  oc get pods -n ${namespace} -o json | jq '.items[] | select(.metadata.name | startswith("${pod_start_with}")) | .metadata.name'
     @{list_pods} =  Split String  ${pod}  \n
     [Return]  ${list_pods}[0]
```

Comment:
Is this actually returning a list of pods? Or is it only the first element of a list?